pulp_smash/tests/rpm/api_v2/test_copy_mtime.py

```diff
@@ -0,0 +1,93 @@
+# coding=utf-8
+"""Test whether copy of directory will copy the ``mtime``."""
+import os
+import re
+import time
+import unittest
+
+from pulp_smash import api, cli, config, selectors, utils
+from pulp_smash.constants import (
+    REPOSITORY_PATH,
+    RPM_SIGNED_URL,
+    RPM_UNSIGNED_FEED_URL,
+)
+from pulp_smash.tests.rpm.api_v2.utils import gen_distributor, gen_repo
+from pulp_smash.tests.rpm.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+_PATH = '/var/lib/pulp/published/yum/https/repos/'
+
+
+class CopyDirectoryStepTestCase(unittest.TestCase):
+    """Test whether copy of directory will copy the ``mtime``."""
+
+    def test_all(self):
+        """Test whether copy of directory will copy the ``mtime``.
+
+        This test targets the following issues:
+
+        * `Pulp Smash #720 <https://github.com/PulpQE/pulp-smash/issues/720>`_
+        * `Pulp #2783 <https://pulp.plan.io/issues/2783>`_
+
+        Do the following:
+
+        1. Create and sync a repository.
+        2. Publish the repository with the option ``generate_sqlite``
+           as True.
+        3. Verify the ``mtime`` for the sqlite files.
+        4. Upload a RPM package to the repository.
+        5. Sync the repository again.
+        6. Assert that ``mtime`` for old sqlite files remain the same.
```

Comment:
Is it necessary to re-publish? Sorry to ask, but I'm more familiar with Satellite terms and flow :)

Reply:
It was not required as part of steps to reproduce the issue. 
Sync in pulp will download the contents of the remote repository and store in our new repository. It is connected to the importer.
Publish will make available for the end user this new content. It is related to the distributor.
I will test if there is any difference, re-publishing before making the assertions.